style.css

```diff
@@ -515,3 +523,365 @@ input[type="text"] {
   font-family: 'Poppins', sans-serif;
   font-style: normal;
 }
+
+@media screen and (min-width: 768px) {
+  .main-header {
+    display: flex;
+    justify-content: right;
+    margin-top: 30px;
+    flex-wrap: wrap;
+    margin-right: 155px;
+  }
+
+  @media screen and (min-width: 768px) {
+    .main-header {
+      display: flex;
+      justify-content: right;
+      margin-top: 30px;
+      flex-wrap: wrap;
+      margin-right: 155px;
+    }
+
+    .desk {
+      display: inline-block;
+      text-decoration: none;
+      font-family: 'Poppins', sans-serif;
+      font-size: 16px;
+      margin-right: 20px;
+      font-style: normal;
+      font-weight: 500;
+      line-height: 20px;
+      color: #344563;
+    }
+
+    .menu {
+      display: none;
+    }
+
+    .nav-container {
+      display: flex;
+      flex-wrap: wrap;
+    }
+
+    .ruby {
+      display: inline-block;
+      display: flex;
+      flex-direction: row;
+      align-items: center;
+      background: #ebebff;
+      border-radius: 8px;
+      color: #6070ff;
+      font-weight: 222px;
+      font-size: 12px;
+      background-color: #ebebff;
+      font-family: 'Poppins', sans-serif;
+      padding: 8px 12px;
+      margin-right: 10px;
+    }
+
+    .part1 {
+      background: url(images/5.png);
+      background-position: cover;
+      background-repeat: no-repeat;
+      background-size: 100% 100%;
+      margin-top: 50px;
+    }
+
+    .head-name {
+      font-family: 'Poppins', sans-serif;
+      font-style: normal;
+      font-weight: 700;
+      font-size: 48px;
+      line-height: 60px;
+      display: flex;
+      align-items: center;
+      letter-spacing: 0.37px;
+      color: #091e42;
+      margin-left: 260px;
+      margin-top: 300px;
+    }
+
+    .head-para {
+      display: flex;
+      width: 920px;
+      font-family: 'Poppins', sans-serif;
+      font-style: normal;
+      font-weight: 400;
+      font-size: 20px;
+      line-height: 28px;
+      color: #344563;
+      margin-left: 260px;
+    }
+
+    .sub-head {
+      margin-left: 260px;
+    }
+
+    .social-media {
+      margin-left: 230px;
+    }
+
+    .logo {
+      margin-left: 155px;
+    }
+
```

Comment:
In the section of your work

- Kindly center your cards as it on the Figma design
- Fix the path to show correctly the images

![image](https://user-images.githubusercontent.com/93630700/229523471-2f1f3f9e-0b85-482b-bf07-6a651a1e1aec.png)